atlas/src/components/ChannelGallery.tsx

```diff
@@ -10,13 +10,17 @@ type ChannelGalleryProps = {
   action?: string
   channels?: ChannelFields[]
   loading?: boolean
+  error?: Error
 }
 
 const PLACEHOLDERS_COUNT = 12
 
-const ChannelGallery: React.FC<ChannelGalleryProps> = ({ title, action, channels, loading }) => {
+const ChannelGallery: React.FC<ChannelGalleryProps> = ({ title, action, channels, loading, error }) => {
   const displayPlaceholders = loading || !channels
 
+  if (error) {
```

Comment:
It doesn't make much sense to pass an error to a component just so it can throw it.

Reply:
Agree, I removed it here and from the Video Gallery